zerver/webhooks/insping/doc.md

```diff
@@ -0,0 +1,53 @@
+Insping (stylized as !nsping) is a simple uptime and performance monitoring
+tool. You'll get to know when the website or service is up and running or down.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+{!append-stream-name.md!}
+
+{!append-topic.md!}
+
+Next, go to [!nsping website](https://www.insping.com). If you haven't created
```

Comment:
Can you use `Insping` here isntead? They "stylised" version is a bit distracting.

Reply:
Probably have to change the display_name in the code PR as well.